### PR TITLE
disco fix

### LINKED ISSUE #2001: gdc disco plot missing mutations
http://localhost:3000/?genome=hg38&dslabel=GDC&disco=1&sample=TARGET-10-PARKFN-09A

should have FLT3 mutation Y589_V592delinsSPG http://localhost:3000/?genome=hg38&gene=flt3&mds3=GDC

### PR DESCRIPTION
**Pull request by xzhou82**
## Description

closes #2001

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [ ] Tests: added and/or passed unit and integration tests, or N/A
- [ ] Todos: commented or documented, or N/A
- [ ] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

**Comment by xzhou82** [comment]
disco must be using a local copy of common.mclass{} and is out of sync because the newly added class is not found. please fix

**Comment by creilly8** [comment]
> 82275eb



> disco must be using a local copy of common.mclass{} and is out of sync because the newly added class is not found. please fix

@xzhou82 This is fixed in https://github.com/stjude/proteinpaint/pull/2005/commits/82275eb6c568bdae3814fc539ef2806f2097f221. Please see [FLT3 rendered here](http://localhost:3000/?genome=hg38&dslabel=GDC&disco=1&sample=TARGET-10-PARKFN-09A).

**Review by creilly8 — APPROVED** [review]

File: client/plots/disco/viewmodel/ViewModelMapper.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export class ViewModelMapper {
 		L: 'exonic',
 		Utr3: 'exonic',
 		Utr5: 'exonic',
+		ProteinAltering: 'exonic',
```
Comment (xzhou82) [inline client/plots/disco/viewmodel/ViewModelMapper.ts:21]: it should not keep a separate static copy of mclasses. can u change?

Comment (aacic) [reply client/plots/disco/viewmodel/ViewModelMapper.ts:21]: This is basically a mapping of which mutations are exonic which are non-exonic. Do we have that somewhere else in the code base or in the DB?

Comment (xzhou82) [reply client/plots/disco/viewmodel/ViewModelMapper.ts:21]: nope. current fix is fine. sorry

**Pull request merged**